docs/tutorials/core-api-mnist-tutorial.rst

```diff
@@ -0,0 +1,93 @@
+.. _core_api_tutorial_part_2:
+
+###########################################################
+Quickstart: Run a Core API MNIST Trial on a Local Cluster
+###########################################################
+
+.. meta::
+  :description: In 5 steps, learn how to integrate the PyTorch MNIST model into Determined AI.
+  :keywords: Core API,MNIST,model developer
+
+In this tutorial, we’ll show you how to integrate a training example with the Determined environment. We'll run our experiment on a local training environment requiring only a single CPU or GPU.
+This tutorial is recommended as an introduction for model developers who are new to Determined AI.
+
+**Objective**
+
+Our goal is to integrate the `PyTorch MNIST training example <https://github.com/pytorch/examples/blob/main/mnist/main.py>`_ into Determined in five steps:
+
+* Start with a bare-bones script
+* Report metrics
+* Perform checkpointing
+* Perform hyperparameter search optimization
+* Run distributed training
+
+**Prerequisites**
+
+* `System Requirements <https://docs.determined.ai/latest/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-prem/requirements.html#system-requirements>`_
+* `Docker <https://docs.determined.ai/latest/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-prem/requirements.html#install-docker>`_
+
+*************************
+Get the Quickstart Files
+*************************
+
+To get started, add the quickstart files to your computer. 
```

Comment:
for now, the quickstart files are in a PR https://github.com/determined-ai/determined/pull/5745